types/react-pdf/dist/Page.d.ts

```diff
@@ -90,6 +90,11 @@ export interface Props {
      */
     noData?: string | React.ReactElement | RenderFunction;
 
+    /**
+     * Function called when page has been clicked.
+     */
+    onClick?: (event: Event) => void;
```

Comment:
It looks like this is called with two arguments, and you can be more specific with the event.
```suggestion
    onClick?: (event: React.MouseEvent, page: PDFPageProxy) => void;
```

Reply:
Thanks for the comment.

I presented more specific with the event.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53843/commits/957f1288c4db7c144705b12c8666210b36332f53